packages/server-wallet/src/wallet/store.ts

```diff
@@ -491,15 +490,15 @@ export class Store {
 
       const objectiveToBeStored: DBObjective = {
         objectiveId: objectiveId(objective),
-        status: 'approved', // TODO: (Stored Objectives) Awkward that it 'auto-approves'... :S
+        status: 'approved',
         type: objective.type,
         participants: [],
         data: {
           targetChannelId,
           fundingStrategy,
         },
       };
-      // TODO: (Stored Objectives) Does it make sense to do the INSERT here?
+
```

Comment:
Why does it make sense to do the INSERT here?

Reply:
This comment was originally added before we used PostgreSQL at all to help @geoknee write the PostgreSQL implementation later and so I wrote it because I wasn't sure if it would be a good place to make a call to an object that needed to be within the context of a database transaction. After we added the database integration, it seemed like the answer was that we _could_ do the insert at this place in the code.